config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
   namespace :api, defaults: { format: 'json' } do
     namespace :v1 do
+      resources :categories, only: %i[index show create update destroy]
```

Comment:
not necessary, since it is not yet specified

Reply:
done